lib/demo_web/controllers/user_confirmation.ex

```diff
@@ -0,0 +1,43 @@
+defmodule DemoWeb.UserConfirmationController do
```

Comment:
I think the path should be `lib/demo_web/controllers/user_confirmation_controller.ex` (it's missing `_controller`).